### PR TITLE
fix: rename form name

### PR DESCRIPTION
**Pull request by ChenRen93**
## Summary
Rename the forms to be able to identify v2 and v3 forms when both are in a configuration file

## Changes

- Forms name
- Test cases selector and file download validator

### CODE REVIEW

File: integration/document-data-upload.spec.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ test("should upload populate data fields correctly for version 2 document", asyn
   await t.expect(ProgressBar.textContent).contains("1");
 
   // Navigate to form
-  await t.click(Button.withText("COO (ChAFTA)"));
+  await t.click(Button.withText("TradeTrust Chafa Certificate of Origin v2"));
```
Comment (simboonlong) [inline integration/document-data-upload.spec.ts:61]: typo, should be ChAFTA

File: integration/document-data-upload.spec.ts [hunk 2]
```diff
@@ -81,22 +81,22 @@ test("should upload populate data fields correctly for version 2 document", asyn
   await t.setFilesToUpload(DataFileDropZoneInput, [dataFileCsvCoo]);
 
   // Validated the content is overwritten by the data file
-  await t.expect(documentNameSelect.innerText).eql("COO-(ChAFTA)-2");
-  await t.expect(fileNameField.value).eql("COO-(ChAFTA)-2");
+  await t.expect(documentNameSelect.innerText).eql("TradeTrust-Chafa-Certificate-of-Origin-v2-2");
+  await t.expect(fileNameField.value).eql("TradeTrust-Chafa-Certificate-of-Origin-v2-2");
```
Comment (simboonlong) [inline integration/document-data-upload.spec.ts:85]: same, think need to find all instances

File: package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "build": "cross-env NODE_OPTIONS='--max-old-space-size=4096' CI=false npm-run-all -s build:css build:js",
     "build:css": "cross-env NODE_ENV=production postcss src/tailwind.css -o src/index.css",
     "build:js": "cross-env NODE_ENV=production react-app-rewired build",
+    "build:config": "ts-node src/test/fixtures/config/configuration-file.tsx",
     "eject": "react-scripts eject",
```
Comment (simboonlong) [inline package.json:23]: the generator stuff got accidentally committed? thought pr is just renaming

Comment (simboonlong) [reply package.json:23]: oh ok this is gg into generator pr ic, ok ignore

File: integration/document-data-upload.spec.ts (hunk 1 above)
Comment (ChenRen93) [reply integration/document-data-upload.spec.ts]: done.

File: package.json (hunk 3 above)
Comment (ChenRen93) [reply package.json:23]: mmmm weird, there is no changes

File: integration/document-data-upload.spec.ts (hunk 2 above)
Comment (ChenRen93) [reply integration/document-data-upload.spec.ts]: done.

**Review by simboonlong — APPROVED** [review]

**Pull request merged**